repose-aggregator/tests/release-verification/src/vagrant/deb/Vagrantfile

```diff
@@ -0,0 +1,172 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+repose_version = ENV['REPOSE_VERSION'] || 'local'
+
+$update_yum = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Updating Apt packages"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+# update package lists from the server
+apt-get update -qq
+#apt-get upgrade -yq
```

Comment:
That makes total sense now. The `upgrade` was not required for testing and caused a major slow down during the boot process. The `update` was needed to force a look at the new repositories and the extra `q` just made it not as noisy. I'll remove the `upgrade` line.